torch_xla/utils/checkpoint_tagger.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import division
+from __future__ import print_function
+
+from collections import defaultdict
```

Comment:
import collections

We use mostly that form in our code.

Reply:
done.